package.json

```diff
@@ -39,7 +39,7 @@
     "test:backend": "pipenv run pytest",
     "test:client": "NODE_ENV=test npm run db:build && npx elm-test",
     "test:review": "npx elm-review",
-    "test:server": "npm run server:build && jest",
+    "test:server": "npm run server:build && NODE_ENV=test jest",
```

Comment:
Why not setting the env var for the whole command? `NODE_ENV=test npm run server:build && jest`

Reply:
because `jest` wouldn't see it, that's the way env var definition scopes work